### ISSUE TITLE
Viewing task properties on the map pans the map away from the feature

### ISSUE THREAD

**Issue by 1ec5** [osmlab/maproulette3#1802]
When you preview a task that represents a feature with many properties, clicking the feature on the map opens a callout bubble and automatically pans the map to fit the callout. Then when you click the Edit button, iD is centered wherever the preview page’s map is currently centered, not where the feature is located, but iD doesn’t show the original feature’s location anymore.

I’ve seen multiple mappers get confused because of the offset. For example, if the point feature was supposed to be on the rooftop of a particular building, but now the map is centered on a different building, the mapper has to manually eyeball the correct building’s location by switching browser tabs. As a workaround, I tell them to click Edit _before_ clicking on the feature to show the callout, or to pan back to the feature before clicking Edit. Actually, my personal workaround is to enable to Task Properties widget and never click on the feature on the map.

I guess it can be useful to open the editor to the current map center, especially for a large non-point feature that the user needs to zoom in on. But maybe there’s a way to meet the user’s expectations in both cases:

* If opening the callout automatically pans the map, then closing the callout should automatically pan the map back to where it was.
* Ideally, the feature would be highlighted in a custom data overlay in iD, so the user doesn’t have to eyeball the center of map even without this panning issue.

**Comment by mvexel** [comment]
You are right to call out that the popup / callout behavior needs to be addressed. Another way to do this is to not show properties in a callout at all, but rather in a separate widget. 

I hadn't considered the data overlay option in iD yet, I don't think I've ever seen it used myself, so that would need some looking into. Intuitively I like it.

**Comment by 1ec5** [comment]
> You are right to call out that the popup / callout behavior needs to be addressed. Another way to do this is to not show properties in a callout at all, but rather in a separate widget.

Yes, I personally use the Task Properties widget, but it can take up a lot of space that might go to waste in challenges that rely on copy buttons in task instructions and the like.

> I hadn't considered the data overlay option in iD yet, I don't think I've ever seen it used myself, so that would need some looking into. Intuitively I like it.

iD accepts a `gpx` parameter that adds the given URL as a custom data layer. The Tasking Manager passes in a GPX file generated by the TM API ([example](https://tm-api.openhistoricalmap.org/api/v2/projects/23/tasks/queries/gpx/?tasks=650)). Despite the parameter name, you can pass in the URL of any file format supported by iD, including GeoJSON. I couldn’t find a MapRoulette API method that returns just the current task’s geometry, so that would need to be added (and would be useful anyhow).

**Comment by Luzandro** [comment]
Have the popups been disabled? I can't see them anymore
